web-app/src/i18n.ts

```diff
@@ -0,0 +1,58 @@
+import i18n from 'i18next';
+import { initReactI18next } from 'react-i18next';
+import Backend from 'i18next-http-backend';
+import LanguageDetector from 'i18next-browser-languagedetector';
+
+export const supportedLanguages = ['en', 'fr'];
+
+i18n
+  .use(Backend)
+  .use(LanguageDetector)
```

Comment:
I tested on the deployed environment, and the language is always detected. The account page is shown on FR even with the feature flag set to false. I suggest removing the detection of the language until we have all the screens translated.